pkg/apis/storage/validation/validation_test.go

```diff
@@ -1229,6 +1271,10 @@ func TestCSINodeValidation(t *testing.T) {
 	}
 }
 
+func makeInt32Ptr(i int32) *int32 {
+	return &i
+}
+
 func TestCSINodeUpdateValidation(t *testing.T) {
```

Comment:
Instead of modifying existing test cases to add allocatable, can we have new test cases for allocatable?

Toggle feature on and off:
- Before no allocatable, try to set allocatable
- Before allocatable, update same allocatable
- Before allocatable, try to change allocatable
- Before allocatable, try to remove allocatable

Reply:
Yeah, I missed a few tests here. Added

What's the difference between `update same (some?) allocatable` and `try to change allocatable`?